packages/backend/index.js

```diff
@@ -46,8 +46,14 @@ app.post("/flocks/:code/votes", (req, res) => {
 	res.send(`Vote added to flock ${req.params.code}`);
 });
 
-app.post("/flocks/:code/basket", (req, res) => {
-	res.send(`Options added to flock ${req.params.code}`);
+app.post("/flocks/:code/basket/:name", async (req, res) => {
```

Comment:
endpoint should probably just be `/flocks/:coop_name/basket` and the egg info can be passed in the request body.